_data-prepper/pipelines/configuration/processors/parse-xml.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: default
+title: parse_xml
+parent: Processors
+grand_parent: Pipelines
+nav_order: 83
+---
+
+# parse_xml
+
+The `parse_xml` processor parses XML data for an event.
+
+## Configuration
+
+You can configure the `parse_xml` processor with the following options.
+
+| Option | Required | Type | Description |
+| :--- | :--- | :--- | :--- | 
+| `source` | No | String | The field in the `event` object that is parsed. Default value is `message`. |
```

Comment:
```suggestion
| `source` | No | String | Tells the processor which field contained in the `event` to parse. Default value is `message`. |
```

Reply:
@natebower: Does this work?

Reply:
Can it just be "Specifies which `event` field to parse"?